exec/cnex/lib/net.c

```diff
@@ -12,7 +12,6 @@
 #include "cell.h"
 #include "exec.h"
 #include "object.h"
-#include "socketx.h"
```

Comment:
What's the purpose of removing this include directive here? Seems that this file doesn't need to be changed at all.

Reply:
Opps.  Fixed this.